### PR TITLE
Use expression-bodies for very simple members.

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**
This was only run on members with the following characteristics:

1. the member header itself must be a single line (i.e. no wrapped parameters).
2. the body of the method must be a single line. (i.e. no complex/wrapped bodies).
3. there must be no non-whitespace trivia in any of the code that is removed.  i.e. no comments/directives/etc.

Furthermore, if the original member+block is on a single line, we preserve that. Otherwise, if the member+block is on multiple lines, then we make a multi-line expression body.

Basically, it must truly be of the form:

```c#
void Foo()
{
    Bar();
}
```

or 

```c#
void Foo() { Bar(); }
```

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
> I do not think merging this pull request would be advantageous.

The advantage here for me is consistency.  I keep jumping from file to file and having totally different styles as i move around.

**Comment by mavasani** [comment]
NOTE: These rules have now been moved to CodeStyle NuGet package, so we can potentially strictly enforce them on CI in Workspaces/Features layers to start with and avoid recurring code cleanup PRs.
